index.html

```diff
@@ -240,5 +240,34 @@ <h3>Skills</h3>
         </li>
       </ul>
     </section>
+
+    <section class="form-container">
+         <div class="form-text">
+           <h1>Contact me</h1>
+         </div>  
+      <div class="paragragh">
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+      </div>
+      
+      <div class="contact-form">
+        <form action="https://formspree.io/f/xzboload" method="post" name="form-collection" class="my-form-collection">
+          <ul>
+            <li>
+                <input type="text" placeholder="benjamin" maxlength="30" required>
+            </li>
+  
+            <li>
+                <input type="text" placeholder="Ben@gmail.com" required>
+            </li>
+  
+            <li>
+                <textarea name="text" cols="30" rows="10" placeholder="write your massage here" maxlength="500"
+                required></textarea>
+            </li>
+          </ul>
+          <button type="submit" class="contact-button">Get in touch</button>
```

Comment:
- [ ] Kindly remove the border for the button as I can't see it in the Figma so you don't need it here and it is better for the user experience


- [ ]  Kindly Fix the width of the button also to match the design 🙏🏻🙏🏻